tests/draft2020-12/optional/format/email.json

```diff
@@ -47,6 +72,16 @@
                 "description": "two subsequent dots inside local part are not valid",
                 "data": "te..st@example.com",
                 "valid": false
+            },
+            {
+                "description": "an invalid domain",
+                "data": "joe.bloggs@invalid=domain.com",
+                "valid": false
+            },
```

Comment:
why is this invalid? My implementation thinks it's fine.

Reply:
A domain is one or more sub-domains separated by '.' (full stop), and a sub domain is letters or digits or the '-' (hyphen-minus) but must start and end only with a letter or a digit.